recipe/meta.yaml

```diff
@@ -14,14 +14,20 @@ build:
   entry_points:
     - retriever = retriever.__main__:main
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
-
+  script: 
+    - python setup.py install --single-version-externally-managed --record=record.txt
```

Comment:
This shouldn't need to change, but if you prefer this style that is fine.

Reply:
I will re-edit it